wasmbinding/queries.go

```diff
@@ -95,6 +95,13 @@ func (qp QueryPlugin) GetDenomAdmin(ctx sdk.Context, denom string) (*bindings.De
 	return &bindings.DenomAdminResponse{Admin: metadata.Admin}, nil
 }
 
+// GetBeforeSendHook is a query to get denom before send hook.
+func (qp QueryPlugin) GetBeforeSendHook(ctx sdk.Context, denom string) (*bindings.BeforeSendHookResponse, error) {
+	cosmWasmAddr := qp.tokenFactoryKeeper.GetBeforeSendHook(ctx, denom)
+
+	return &bindings.BeforeSendHookResponse{CosmWasmAddr: cosmWasmAddr}, nil
```

Comment:
I personally don't think so. if hook is not set, it is not an error as soon as it's not obligatory to have such hook for every denom. it could be empty (and it's default behaviour)

Reply:
no reason for the `GetBeforeSendHook` to have an error variable in return definition then